content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/postprocess.rb

```diff
@@ -0,0 +1,34 @@
+#
+# Description: This method examines the orchestration stack provisioned
+#
+module ManageIQ
+  module Automate
+    module Service
+      module Generic
+        module StateMachines
+          module GenericLifecycle
+            class PostProcess
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                @handle.log("info", "Starting PostProcessing")
+
+                task = @handle.root["service_template_provision_task"]
```

Comment:
@tinaafitz Should we add a 2 line method called get_service and raise an exception if service is not found